Docs: add a WeChat QR code (#2351)
oam-dev/kubevela: README.md

- Slack:  [CNCF Slack](https://slack.cncf.io/) #kubevela channel (*English*)
- Gitter: [oam-dev](https://gitter.im/oam-dev/community) (*English*)
- [DingTalk Group](https://page.dingtalk.com/wow/dingtalk/act/en-home): `23310022` (*Chinese*)

> - Wechat Group (*Chinese*) : Add developer‘s wechat to join the user group.

Reviewer: ```suggestion
- Wechat Group (*Chinese*): Broker wechat to add you into the user group.
```
